### DISCUSSION TITLE
Task icon

### DISCUSSION THREAD

**Discussion by robyn-bautista** [twilio-labs/paste#3485] [Request: Icon] [answered]
### Description

Hi there - we need an updated task icon for our channel list in flex that comes from paste. Are there any suggestions on which to use? In the past, we've used this icon 
![image](https://github.com/twilio-labs/paste/assets/123405403/ede340cd-d19f-4dbe-8993-bfe016ff5e7b)

The paste product icon for "ContactCenterAssessments" works well, but there isn't a correlated one on the UI icon list

<img width="52" alt="Screenshot 2023-09-13 at 12 25 08 PM" src="https://github.com/twilio-labs/paste/assets/123405403/1e855136-ddac-4913-b261-d4c0b2f6cc11">

Here is a list of all channel types for context: https://www.figma.com/file/O8WodgwGI9jB9aXkD1eQND/Standards-of-Flex?type=design&node-id=59%3A4443&mode=design&t=dql4RP94U65fY3ec-1

### Link to icon

n/a

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment]
Hey @robyn-bautista ! For some context to help figure out the best option:
1. How do these unknown/default tasks get created?
2. Does something like the "Notes" icon work for this case, or does it stray too far from its meaning? Just want to make sure an existing icon doesn't work just as well before we create a new one.
3. Are tasks related to assessments at all?

**Comment by robyn-bautista** [comment]
1. If a task channel was selected as "Default" in studio flow by our customer, then none of our other channel configurations are applicable (SMS, voice, etc). A customer might do this so they can configure different types of tasks - this can range from reminders to other actions/tasks they might create on their own. A better label for this kind of task might be "generic task" (we call it default on the backend)

2. We thought about using notes - but we're using that icon for when an agent creates "notes" on a task. Basically when an agent takes notes on any kind of task whether voice, email, etc.

3. They are not related to assessments as far as I know

**Reply by serifluous** [reply]
sounds good! The glyph you shared probably works fine if y'all have already used it in the past. Can you fill out the dependency tracker form? https://airtable.com/appwvsQKyJHfnG5Sa/shrvMvL3BG6Zm0f9l

**Comment by robyn-bautista** [comment]
Sounds good, I'll fill out the form - Thanks Sarah!

**Reply by serifluous** [reply]
Just published the icon in Figma. I tagged you and Stu on the eng ticket.

**Reply by serifluous** [reply, marked as answer]
Just published the icon in Paste: https://paste.twilio.design/components/icon

**Reply by robyn-bautista** [reply]
amazing, thank you!
